mgmtsystem_action/models/mgmtsystem_action_stage.py

```diff
@@ -1,21 +1,59 @@
 # -*- coding: utf-8 -*-
+# Copyright (C) 2010 Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from openerp import models, fields
+from odoo import fields, models
 
 
 class MgmtsystemActionStage(models.Model):
-    """This object is used to defined different stage for actions."""
 
     _name = 'mgmtsystem.action.stage'
-    _description = "Action stages"
+    _description = "Action Stage"
     _order = 'sequence'
 
-    name = fields.Char('Stage Name', required=True, translate=True)
-    sequence = fields.Integer(
-        'Sequence', help="Used to order stages. Lower is better.", default=100)
-    case_default = fields.Boolean(
-        'Common to All Teams', help="If you check this field, \
-        this stage will be proposed by default on each sales team. \
-        It will not assign this stage to existing teams.")
-    is_starting = fields.Boolean('Starting stage')
+    def _get_mail_template_id_domain(self):
+        return [('model', '=', 'mgmtsystem.action')]
+
+    name = fields.Char(string='Stage Name', required=True, translate=True)
+    case_default = fields.Boolean('Common to All Teams')
+    is_starting = fields.Boolean(
+        string='Is starting Stage',
+        help="select stis checkbox if this is the default stage \n"
```

Comment:
"Select this checkbox if this is default stage." (it might for for something else than nonconformities)